pkg/kubelet/dockershim/docker_image.go

```diff
@@ -51,7 +52,7 @@ func (ds *dockerService) ListImages(_ context.Context, r *runtimeapi.ListImagesR
 	for _, i := range images {
 		apiImage, err := imageToRuntimeAPIImage(&i)
 		if err != nil {
-			// TODO: log an error message?
+			glog.Warningf("Failed to convert docker API image to runtime API image: %v", err)
```

Comment:
It'd help to print the docker image as well. 
Also make it a V(5) Info to avoid spamming the log on every listing

Reply:
@yujuhong Done